test/test_nn.py

```diff
@@ -1409,6 +1409,7 @@ def test(nonlinearity, *args, **kwargs):
         test('log_softmax', 0)
         test('leaky_relu', 0.2)
 
+    @unittest.skip('testing')
```

Comment:
Are relus failing now if we pass in a nan tensor?

Reply:
This is about properly propagating NaNs. If you pass a NaN into the layer, do you get a NaN out. I'll need to look into this more closely.